### ISSUE TITLE
Le constructeur RandomVector(Function, RandomVector) est obsolète avec openturns>=1.12

### ISSUE THREAD

**Issue by adumasphi** [persalys/persalys#126]
In GitLab by @ladier on Oct 1, 2018, 11:04

Il faut utiliser CompositeRandomVector cf https://github.com/openturns/openturns/pull/858, remplacer par exemple dans ces lignes:
```
lib/src/base/SORMAnalysis.cxx:75:  Event event(RandomVector(function, getPhysicalModel().getInputRandomVector()), getLimitState().getOperator(), getLimitState().getThreshold());
lib/src/base/SimulationReliabilityAnalysis.cxx:154:  Event event(RandomVector(function, getPhysicalModel().getInputRandomVector()), getLimitState().getOperator(), getLimitState().getThreshold());
lib/src/base/FORMAnalysis.cxx:75:  Event event(RandomVector(function, getPhysicalModel().getInputRandomVector()), getLimitState().getOperator(), getLimitState().getThreshold());

```

**Comment by adumasphi** [comment]
In GitLab by @schueller on Oct 2, 2018, 12:01

changed title from **{-test creation ticket-}** to **{+Le constructeur RandomVector(Function, RandomVector) est obsolète avec openturns>=1.12+}**

**Comment by adumasphi** [comment]
In GitLab by @schueller on Oct 2, 2018, 12:01

changed the description

**Comment by adumasphi** [comment]
In GitLab by @schueller on Oct 2, 2018, 12:02

changed the description

**Comment by adumasphi** [comment]
In GitLab by @schueller on Oct 4, 2018, 09:24

mentioned in commit 9c9931a91bec18d3be31691756e599ee96cc7a10

**Comment by adumasphi** [comment]
In GitLab by @schueller on Oct 5, 2018, 18:07

mentioned in commit f3083781d73da4b526a8ea56f75de5e8e8d16b69

**Comment by adumasphi** [comment]
In GitLab by @schueller on Oct 5, 2018, 18:07

closed via commit 9c9931a91bec18d3be31691756e599ee96cc7a10

**Comment by adumasphi** [comment]
In GitLab by @schueller on Oct 5, 2018, 18:07

closed via merge request !209
